### PR TITLE
Add link to github roadmap to primary and footer navigation

### PR DESCRIPTION
**Pull request by insectengine**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
Good idea!

File: _includes/header-navigation.html [hunk 1]
```diff
@@ -40,10 +40,11 @@
           <li><a href="{{site.baseurl}}/insights" class="{% if page.url contains '/insights/' %}active{% endif %}">PODCAST</a></li>
           <li><a href="{{site.baseurl}}/events" class="{% if page.url contains '/events/' %}active{% endif %}">EVENTS</a></li>
           <li><a href="{{site.baseurl}}/newsletter" class="{% if page.url contains '/newsletter/' %}active{% endif %}">NEWSLETTER</a></li>
+          <li><a href="https://github.com/orgs/quarkusio/projects/13/views/1" class="{% if page.url contains '/roadmap/' %}active{% endif %}">ROADMAP</a></li>
```
Comment (gastaldi) [inline _includes/header-navigation.html:43]: I thought it would use what's defined in the `_data/projectfooter.yaml`? Why repeat here?

File: _includes/header-navigation.html [hunk 2]
```diff
@@ -40,10 +40,11 @@
...
           </ul>
       </li>
       <li>
-        <a href="https://code.quarkus.io" class="button-cta secondary white">Start Coding</a>
+        <a href="https://code.quarkus.io" class="button-cta secondary white">START CODING</a>
```
Comment (gastaldi) [inline _includes/header-navigation.html:47]: Is this change related to this PR?

Comment (insectengine) [reply _includes/header-navigation.html:47]: Yes it was intentional but it's more of just a tidying up of the HMTL since the CSS transforms to uppercase. All of the rest of the links are in all caps before CSS.

**Comment by gsmet** [comment]
@gastaldi I let you merge it if you think it's OK?

**Comment by gastaldi** [comment]
It looks good to me. Let's merge it!
